src/cert_store.rs

```diff
@@ -243,6 +243,63 @@ impl CertStore {
             Ok(ret)
         }
     }
+
+    /// Determines if a cert exists in a given store
+    pub fn find_existing_cert(&mut self, cert: &CertContext) -> io::Result<CertContext> {
+        unsafe {
+            let res = Cryptography::CertFindCertificateInStore( 
+                self.0,
+                Cryptography::X509_ASN_ENCODING | 
+                Cryptography::PKCS_7_ASN_ENCODING, 
```

Comment:
Formatting: indent or include in the above line.

Reply:
Not sure what you mean here, indenting looks fine to me